### PR TITLE
Added an example for DotNet bulk subscribing

### PR DESCRIPTION
**Pull request by sab-codes**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [x] N/A - Commands include options for Linux, MacOS, and Windows within codetabs
- [x] N/A - New file and folder names are globally unique
- [x] N/A - Page references use shortcodes instead of markdown or URL links
- [x] N/A - Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

The bulk pub/sub documentation is missing the DotNet bulk subscribing example so I added an example.

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md [hunk 1]
```diff
@@ -334,7 +334,7 @@ Please refer [Expected HTTP Response for Bulk Subscribe]({{< ref pubsub_api.md >
 
 Please refer following code samples for how to use Bulk Subscribe:
 
-{{< tabs Java Javascript "HTTP API (Bash)" "HTTP API (PowerShell)" >}}
+{{< tabs Java Javascript DotNet "HTTP API (Bash)" "HTTP API (PowerShell)" >}}
```
Comment (hhunter-ms) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md]:
```suggestion
{{< tabs Java JavaScript ".NET" "HTTP API (Bash)" "HTTP API (PowerShell)" >}}
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md (hunk 1 above)
Comment (sab-codes) [reply daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md]: @hhunter-ms Thank you for the suggestion! I do like .NET better, but to be consistent with the rest of the documentation, I believe it should mirror what is on the rest of the pages in the documentation including the code tabs at the top of the bulk pub/sub. That being said I did have a case difference from what is on [line 25](https://github.com/dapr/docs/blob/570700769c1abbf449d55959235ddf9293ccb860/daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md?plain=1#L25).
```suggestion
{{< tabs Java Javascript Dotnet "HTTP API (Bash)" "HTTP API (PowerShell)" >}}
```

File: daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md [hunk 2]
```diff
@@ -334,7 +334,7 @@ Please refer [Expected HTTP Response for Bulk Subscribe]({{< ref pubsub_api.md >
 
 Please refer following code samples for how to use Bulk Subscribe:
 
-{{< tabs Java Javascript "HTTP API (Bash)" "HTTP API (PowerShell)" >}}
+{{< tabs Java Javascript Dotnet "HTTP API (Bash)" "HTTP API (PowerShell)" >}}
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-bulk.md]:
```suggestion
{{< tabs "Java" "JavaScript" ".NET" "HTTP API (Bash)" "HTTP API (PowerShell)" >}}
```

**Comment by hhunter-ms** [comment]
@halspang do you mind taking a look at this code example for bulk subscribe?

**Pull request merged**